### PR TITLE
Homework 5

### PR DESCRIPTION
**Pull request by Temurbtz**

### CODE REVIEW

File: cleaning_system/cleaning_company/views.py [hunk 1]
```diff
@@ -1,3 +1,48 @@
...
-# Create your views here.
+from .models  import Company
+from .serializers  import CompanySerializer
+from rest_framework.decorators import api_view
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework.parsers import JSONParser
+from rest_framework.response import Response
+from django.http import  JsonResponse
+from rest_framework import status
+@api_view(['GET', 'POST'])
```
Comment (Temurbtz) [inline cleaning_system/cleaning_company/views.py:10]: Created  view  with  api_view  decorator  for company  model

File: cleaning_system/cleaning_company/serializers.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+from rest_framework import serializers
+from  .models  import Company
+from service_type.models import Service
+import  datetime
+
```
Comment (Temurbtz) [inline cleaning_system/cleaning_company/serializers.py:5]: Added   serializer  for  company  model

File: cleaning_system/rating_review/serializers.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+from rest_framework import serializers
+from  .models  import  Review
+from  cleaning_company.models  import Company
+from django.contrib.auth.models import User
+from  cleaning_company.models  import  Company
+
+
+
```
Comment (Temurbtz) [inline cleaning_system/rating_review/serializers.py:8]: Created  serializer  for  review  model

File: cleaning_system/rating_review/views.py [hunk 4]
```diff
@@ -1,2 +1,45 @@
-from django.shortcuts import render
+from django.http import Http404
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
 
+
```
Comment (Temurbtz) [inline cleaning_system/rating_review/views.py:6]: Created  view  for  review  model  with class  APIview

File: cleaning_system/request/views.py [hunk 5]
```diff
@@ -1,3 +1,48 @@
-from django.shortcuts import render
+from django.http import Http404
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
 
-# Create your views here.
```
Comment (Temurbtz) [inline cleaning_system/request/views.py]: Created  views  for request  model  with APIview

File: cleaning_system/request/serializers.py [hunk 6]
```diff
@@ -0,0 +1,38 @@
+from rest_framework import serializers
+from cleaning_company.models import Company
+from   service_type.models  import Service
+from .models  import Request
```
Comment (Temurbtz) [inline cleaning_system/request/serializers.py:4]: Serializer  for  request  model

File: cleaning_system/service_type/serializers.py [hunk 7]
```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+from .models import Service
+
```
Comment (Temurbtz) [inline cleaning_system/service_type/serializers.py:3]: Serializer  for  service  model

File: cleaning_system/service_type/views.py [hunk 8]
```diff
@@ -1,3 +1,45 @@
-from django.shortcuts import render
 
-# Create your views here.
+
+from django.http import HttpResponse, JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework.parsers import JSONParser
+from  .models  import Service
+from  .serializers  import ServiceSerializer
+from rest_framework.decorators import api_view
```
Comment (Temurbtz) [inline cleaning_system/service_type/views.py:8]: Created  views  for service  model  with  apiview  decorator

**Review by DennTerentyev — APPROVED** [review]

**Pull request merged**